internal/ethapi/api.go

```diff
@@ -88,6 +88,7 @@ type feeHistoryResult struct {
 	GasUsedRatio []float64        `json:"gasUsedRatio"`
 }
 
+// FeeHistory function
```

Comment:
Nice that you added a comment, but it's not a very helpful one

Reply:
Please remove this or write something sensible

Reply:
nevermind, I fixed it